neuron_explainer/models/autoencoder.py

```diff
@@ -93,8 +93,14 @@ def from_state_dict(
         n_latents, d_model = state_dict["encoder.weight"].shape
         autoencoder = cls(n_latents, d_model)
         autoencoder.load_state_dict(state_dict, strict=strict)
+        autoencoder.activation = state_dict.get("activation", nn.ReLU())
```

Comment:
is strict=strict on the line above going to be an issue?

maybe 
```
autoencoder.activation = state_dict.pop("activation", nn.ReLU())
autoencoder.load_state_dict(state_dict, strict=strict)
```

Reply:
also do you think we should just have strings + a mapping to modules?  worry a bit about weird things with pickling modules

Reply:
huh, i guess this is how it is done in our actual codebase also

Reply:
I removed the pickling